src/components/explorer/IndividualPhenotypicFeatures.js

```diff
@@ -20,7 +20,7 @@ const PHENOTYPIC_FEATURES_COLUMNS = [
                 </h4>
             ) : <>
                 <OntologyTerm term={type} />{" "}
-                {negated ? (
+                {excluded ? (
```

Comment:
can you also add this property (`excluded`) to `propTypes.js` (which is barely used, but just for completeness?)

Reply:
this is a good point, I already ran across a [propTypes issue](https://206.12.92.46/issues/2069) in testing this pr